src/proc/formats-converter.h

```diff
@@ -26,6 +26,8 @@ namespace librealsense
                                         std::function< std::shared_ptr< processing_block >( void ) > generate_func );
         void register_converter( const processing_block_factory & pbf );
         void register_converters( const std::vector< processing_block_factory > & pbfs );
+        void clear_registered_converters();
+        void drop_non_basic_formats();
```

Comment:
I don't see any documentation as to what "basic" formats are. I pretty much know what you mean, but maybe we need a more central spot that documents what `use-basic-formats` is?

[rs_context.h](https://github.com/IntelRealSense/librealsense/blob/dds/include/librealsense2/h/rs_context.h#L35) has the context stuff listed. Maybe it needs updating?

Reply:
Good idea, I will update there, and also add comments in formats_converter

Reply:
Added comments